source/Ship.cpp

```diff
@@ -3361,6 +3361,7 @@ double Ship::JumpFuelMissing() const
 // Get the heat level at idle.
 double Ship::IdleHeat() const
 {
+	/*
```

Comment:
I think this is just because you wanted to try something out, but please delete the comment block that is not needed.